standard/info/wikis/geoguessr/info.lua

```diff
@@ -0,0 +1,42 @@
+---
+-- @Liquipedia
+-- wiki=geoguessr
+-- page=Module:Info
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+return {
+	startYear = 2013,
+	wikiName = 'geoguessr',
+	name = 'GeoGuessr',
+	defaultGame = 'geoguessr',
+	games = {
+		geoguessr = {
+			abbreviation = 'Geo',
```

Comment:
The abbr was just changed, was that intended?

Reply:
this should be corrected now as the full game name being the abbreviation (game dont really have proper abbr, will just use the full name)